docs/Getting Started/Integration Basics/index.md

```diff
@@ -811,7 +823,47 @@ The `sdlManager` must be shutdown properly if this class is shutting down in the
 !@
 
 @![android,javaSE,javaEE]
-### Determining SDL Support
+#### Optional SdlManager Builder Parameters
+!@
+
+@![android,javaSE,javaEE]
```

Comment:
These are the same, so the lower one can be removed

Reply:
Done!